less/about/preferences.less

```diff
@@ -247,8 +247,6 @@ input[type="checkbox"][disabled] {
 }
 
 table.sortableTable {
-  width: 704px;
```

Comment:
`width: 704px` is used on `about:preferences#search` only so replaced it with `sortableTable_searchTab`. Seea above.